internal-docs/onboarding/001-Vadman97.md

```diff
@@ -0,0 +1,82 @@
+<!--- 
+Instructions for onboarding buddy:
+
+* cp 000-template.md 001-github-new-hire-username.md
+* Make appropriate changes to this template given who you are onboarding.
+-->
+
+# Welcome [@Vadman97](https://github.com/Vadman97) 👋
+
+Welcome to Highlight [@Vadman97](https://github.com/Vadman97)! We're thrilled to have you join us.
+Recently, our company went [open source](https://news.ycombinator.com/item?id=34897645) and we are [building our work in public](https://www.highlight.io/docs/general/company/values#we-build-in-public).
+
+Your onboarding buddy is [@et](https://github.com/et) and they'll walk you through your next couple weeks.
+
+## Feedback
+
+Highlight is a fast moving start up that is constantly evolving. We definitely don't have everything right.
+
+You are absolutely encouraged to edit the company at any time either through a pull request or through a public slack channel.
+
+# Timeline
+
+## Ahead of day one
+
+Prior to joining Highlight, ensure you have ordered a laptop that will arrive before your first day (work with Jay or your onboarding buddy to figure this out).
+
+### Ahead of day one access checklist
+
+* Google Workspace
+* Slack
+
+## Week one
+
+Your onboarding buddy will help you through the following tasks over your first week but prioritize getting to know the team and understanding the product over delivering work. 
+
+- [ ] https://github.com/highlight/highlight/issues/4933
+
+### Day one
+
+* Have informal 1-1s with everyone on the team (or have them scheduled)
+* Get your [development environment](https://www.highlight.io/docs/getting-started/self-host/self-hosted-hobby-guide) set up
+* Pick up one of the tasks above.
+
+### Day two (dogfooding exercise)
+
+- Start at Highlight's [homepage](https://www.highlight.io/), find the sign up page to sign up with your personal email.
+- Set up a frontend app in your framework of choice and integrate Highlight [using our docs](https://www.highlight.io/docs/getting-started/overview) (or the setup page) to record a session.
+- Set up a backend app in a language/framework of your choice and record errors and logs.
+- Give us feedback on the process.
+    - Where did you experience friction?
+    - Were the docs incorrect anywhere?
+    - Were the docs easy to navigate?
+    - Was the API spec in the expected style to the language? (e.g. naming made sense or matched the stylistic nature of the language).
+
+### End of week one access checklist
+
+* Highlight
+* GitHub
+* Zoom
+* Loom
+* Figma
+* Doppler
+* Notion (much of this outdated)
+
+## Week two
+
+Set up pairing with each member of the team on the following tasks:
+
+- [ ] https://github.com/highlight/highlight/issues/4764 with [@ccschmitz](https://github.com/ccschmitz)
+- [ ] https://github.com/highlight/highlight/issues/4837 with [@mayberryzane](https://github.com/mayberryzane)
+
+### End of week two access checklist
+
+* Render
+* Datadog
```

Comment:
Can we stick all the access parts under a single heading? Imo, it might be confusing to have to scroll through this doc to see what I do/don't have access to. Within that single list, we could sort it by importance maybe?